### DISCUSSION TITLE
Doris BE failed of 2.0.4 version

### DISCUSSION THREAD

**Discussion by lc3caa** [apache/doris#30883] [A - General / Q&A]
报错日志如下
*** Query id: 94f65d3b85244e83-a820c704967cd485 ***
*** tablet id: 0 ***
*** Aborted at 1707148148 (unix time) try "date -d @1707148148" if you are using GNU date ***
*** Current BE git commitID: 003a815b63 ***
*** SIGABRT unknown detail explain (@0x5a0c) received by PID 23052 (TID 23698 OR 0x7f92751b2700) from PID 23052; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/src/common/signal_handler.h:417
 1# 0x00007F93B68BC400 in /lib64/libc.so.6
 2# __GI_raise in /lib64/libc.so.6
 3# abort in /lib64/libc.so.6
 4# __gnu_cxx::__verbose_terminate_handler() [clone .cold] at ../../../../libstdc++-v3/libsupc++/vterminate.cc:75
 5# __cxxabiv1::__terminate(void (*)()) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
 6# 0x00005608B40D7AD1 in /data/doris/be/lib/doris_be
 7# 0x00005608B40D7C24 in /data/doris/be/lib/doris_be
 8# doris::vectorized::ColumnString::insert_data(char const*, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/vec/columns/column_string.h:187
 9# doris::vectorized::ColumnNullable::insert_many_strings(doris::StringRef const*, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/vec/columns/column_nullable.cpp:213
10# _ZNSt8__detail9__variant17__gen_vtable_implINS0_12_Multi_arrayIPFNS0_21__deduce_visit_resultIvEEOZN5doris10vectorized15AggregationNode47_serialize_with_serialized_key_result_non_spillEPNS5_12RuntimeStateEPNS6_5BlockEPbE3$_0RSt7variantIJNS6_27AggregationMethodSerializedI9PHHashMapINS5_9StringRefEPc11DefaultHashISI_vELb0EEEENS6_26AggregationMethodOneNumberIh12FixedHashMapIhSJ_28FixedHashMapImplicitZeroCellIhSJ_16HashTableNoStateE28FixedHashTableCalculatedSizeISS_E9AllocatorILb1ELb1ELb0EEELb0EEENSO_ItSP_ItSJ_SQ_ItSJ_SR_E24FixedHashTableStoredSizeISZ_ESW_ELb0EEENSO_IjSH_IjSJ_9HashCRC32IjELb0EELb0EEENSO_ImSH_ImSJ_S14_ImELb0EELb0EEENS6_30AggregationMethodStringNoCacheI13StringHashMapISJ_SW_EEENSO_INS6_7UInt128ESH_IS1F_SJ_S14_IS1F_ELb0EELb0EEENSO_IjSH_IjSJ_14HashMixWrapperIjS15_ELb0EELb0EEENSO_ImSH_ImSJ_S1J_ImS18_ELb0EELb0EEENSO_IS1F_SH_IS1F_SJ_S1J_IS1F_S1G_ELb0EELb0EEENS6_37AggregationMethodSingleNullableColumnINSO_IhNS6_26AggregationDataWithNullKeyISX_EELb0EEEEENS1T_INSO_ItNS1U_IS12_EELb0EEEEENS1T_INSO_IjNS1U_IS16_EELb0EEEEENS1T_INSO_ImNS1U_IS19_EELb0EEEEENS1T_INSO_IjNS1U_IS1L_EELb0EEEEENS1T_INSO_ImNS1U_IS1O_EELb0EEEEENS1T_INSO_IS1F_NS1U_IS1H_EELb0EEEEENS1T_INSO_IS1F_NS1U_IS1R_EELb0EEEEENS1T_INS1B_INS1U_IS1D_EEEEEENS6_26AggregationMethodKeysFixedIS19_Lb0EEENS2M_IS19_Lb1EEENS2M_IS1H_Lb0EEENS2M_IS1H_Lb1EEENS2M_ISH_INS6_7UInt256ESJ_S14_IS2R_ELb0EELb0EEENS2M_IS2T_Lb1EEENS2M_IS1O_Lb0EEENS2M_IS1O_Lb1EEENS2M_IS1R_Lb0EEENS2M_IS1R_Lb1EEENS2M_ISH_IS2R_SJ_S1J_IS2R_S2S_ELb0EELb0EEENS2M_IS31_Lb1EEEEEEJEEESt16integer_sequenceImJLm18EEEE14__visit_invokeESE_S35_ at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/variant:1013
11# doris::vectorized::AggregationNode::_serialize_with_serialized_key_result_non_spill(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /home/zcp/repo_center/doris_release/doris/be/src/vec/exec/vaggregation_node.cpp:1642
12# doris::vectorized::AggregationNode::_serialize_with_serialized_key_result(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /home/zcp/repo_center/doris_release/doris/be/src/vec/exec/vaggregation_node.cpp:1528
13# std::_Function_handler<doris::Status (doris::RuntimeState*, doris::vectorized::Block*, bool*), std::_Bind_result<doris::Status, doris::Status (doris::vectorized::AggregationNode::*(doris::vectorized::AggregationNode*, std::_Placeholder<1>, std::_Placeholder<2>, std::_Placeholder<3>))(doris::RuntimeState*, doris::vectorized::Block*, bool*)> >::_M_invoke(std::_Any_data const&, doris::RuntimeState*&&, doris::vectorized::Block*&&, bool*&&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
14# doris::vectorized::AggregationNode::pull(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /home/zcp/repo_center/doris_release/doris/be/src/vec/exec/vaggregation_node.cpp:590
15# doris::vectorized::AggregationNode::get_next(doris::RuntimeState*, doris::vectorized::Block*, bool*) in /data/doris/be/lib/doris_be
16# std::_Function_handler<doris::Status (doris::RuntimeState*, doris::vectorized::Block*, bool*), std::_Bind<doris::Status (doris::ExecNode::*(doris::ExecNode*, std::_Placeholder<1>, std::_Placeholder<2>, std::_Placeholder<3>))(doris::RuntimeState*, doris::vectorized::Block*, bool*)> >::_M_invoke(std::_Any_data const&, doris::RuntimeState*&&, doris::vectorized::Block*&&, bool*&&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
17# doris::ExecNode::get_next_after_projects(doris::RuntimeState*, doris::vectorized::Block*, bool*, std::function<doris::Status (doris::RuntimeState*, doris::vectorized::Block*, bool*)> const&, bool) at /home/zcp/repo_center/doris_release/doris/be/src/exec/exec_node.cpp:603
18# doris::PlanFragmentExecutor::get_vectorized_internal(doris::vectorized::Block*, bool*) at /home/zcp/repo_center/doris_release/doris/be/src/runtime/plan_fragment_executor.cpp:354
19# doris::PlanFragmentExecutor::open_vectorized_internal() at /home/zcp/repo_center/doris_release/doris/be/src/runtime/plan_fragment_executor.cpp:319
20# doris::PlanFragmentExecutor::open() at /home/zcp/repo_center/doris_release/doris/be/src/runtime/plan_fragment_executor.cpp:271
21# doris::FragmentExecState::execute() at /home/zcp/repo_center/doris_release/doris/be/src/runtime/fragment_mgr.cpp:267
22# doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::RuntimeState*, doris::Status*)> const&) at /home/zcp/repo_center/doris_release/doris/be/src/runtime/fragment_mgr.cpp:549
23# std::_Function_handler<void (), doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::RuntimeState*, doris::Status*)> const&)::$_0>::_M_invoke(std::_Any_data const&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
24# doris::ThreadPool::dispatch_thread() in /data/doris/be/lib/doris_be
25# doris::Thread::supervise_thread(void*) at /home/zcp/repo_center/doris_release/doris/be/src/util/thread.cpp:499
26# start_thread in /lib64/libpthread.so.0
27# clone in /lib64/libc.so.6

**Comment by yagagagaga** [comment]
Can you provide the table schema and the sql?
